Diagrams/readme.md

```diff
@@ -0,0 +1,9 @@
+# Diagrams
+
+## C4 Diagrams
+
+A simple overview of the structure of this project is maintained using C4 diagrams. These can be edited [here](https://lucid.app/lucidspark/5b53254d-274b-4512-9285-31eaf3057069/edit?viewport_loc=-3479%2C-2061%2C10362%2C5564%2C0_0&invitationId=inv_5ccb3d50-1ce8-411e-8ccb-4afb2b46af19)
```

Comment:
This link should be read only... if it's not big whoops it probably shouldn't be commited to a public repo 😆